lib/tools/apk-utils.js

```diff
@@ -473,13 +473,79 @@ apkUtilsMethods.getDeviceLocale = async function () {
 };
 
 /**
- * Set the locale name of the device under test.
+ * Set the locale name of the device under test and the format of the locale is en-US, for example.
+ * This method call setDeviceLanguageCountry, so, please use setDeviceLanguageCountry as possible.
  *
- * @param {string} locale - The name of the new device country.
+ * @param {string} locale - The name of device country connected with `-`. e.g. en-US.
  */
 apkUtilsMethods.setDeviceLocale = async function (locale) {
-  // this method is only used in API >= 23
-  await this.setDeviceSysLocale(locale);
+  let split_locale = locale.split("-");
```

Comment:
can we add some input validation here?

Reply:
at least that the string has a dash, so split_locale[0] won't raise an unexpected exception